sdk/core/azure-mgmt-core/CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 
 # Release History
 
-## 1.2.1 (unreleased)
+## 1.2.1 (Unreleased)
```

Comment:
Shouldn't be 1.3.0?

Reply:
i went with a minor version bump because I see this as a bug fix, instead of a new feature

Reply:
Is "path_format_arguments" supported in 1.2.0? If not, let's move it to 1.3.0.

Reply:
ok, i'll move mgmt core to 1.3.0 and azure core to 1.9.0